src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,20 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeMoreInfoAboutEqualShareModal();
+    }
+
+    public void closeMoreInfoAboutEqualShareModal() {
+        WebElement closeButton = browser.driver.findElement(By
+                .xpath("//div[@id='more-info-equal-share-modal']//button[@class='close']"));
```

Comment:
Looks like this page has a lot of similar repetitions. We need to look at the whole page and refactor out the utility methods, which can be done as a separate issue. In the meantime, you can take one of the three options for this PR.